stable/sealed-secrets/templates/configmap-dashboards.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.dashboards.create }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "sealed-secrets.fullname" . }}-controller-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    grafana_dashboard: "1"
```

Comment:
Please add the possibility to define custom labels (like in the ServiceMonitor).